### DISCUSSION TITLE
Effect alert blink

### DISCUSSION THREAD

**Discussion by RudySixOw** [Koenkk/zigbee2mqtt#5864] [General]
Hello,

I'm trying to play with the effects to make alerts, and I can't find a way to apply colors to them.
examples:
{"color": {"x": 0.6923, "y": 0.2955}} {"effect": "blink"}
{"color": {"hex": "# 001EFF"}, "alert": "lselect"}
{"color": {"hex": "# 001EFF"}, "effect": "breathe"}

I can't find the subtlety to make this work.
If anyone knows the solution thanks in advance.

**Comment by Koenkk** [comment]
Colors are not supported for effects (this is not included in the Zigbee spec)

**Reply by RudySixOw** [reply]
ok thanks for the quick answer.
Too bad this would have made it possible to distinguish different types of alert for example flooding, smoke detection ...
